### DISCUSSION TITLE
Idea: Medium shield

### DISCUSSION THREAD

**Discussion by adamlui** [badges/shields#10013] [Ideas]
Medium is quite popular w/ over 100 million monthly readers as of 2022, a follower shield would be very nice and useful to many probably

**Comment by jNullj** [comment]
Looks like the Medium api is depricated, do you have an idea for how to extract this data?
https://github.com/Medium/medium-api-docs

There is https://mediumapi.com/ but the limit for free plan is 150 calls/mo

**Reply by adamlui** [reply]
There is a shields.io collaborator that reached out to YouTube on multiple occasions to increase limit before, if you could build a shield using this free one, perhaps he could work his diplomatic magic to raise Medium's limit, too

**Reply by adamlui** [reply]
Oh wait I didn't realize that was unofficial, IDK then 🙁

**Reply by adamlui** [reply]
...because I dont think the free API owner has incentive to give away requests
